examples/static_graph_optimizations/cifar/README.md

```diff
@@ -0,0 +1,26 @@
+# Convolutional neural networks for CIFAR-10 and CIFAR-100 Classification
```

Comment:
Add `with static subgraph optimization` to make clear how this is different from the other one.